src/app/content/routes.ts

```diff
@@ -19,7 +18,7 @@ interface State {
   bookUid: string;
   bookVersion: string;
   pageUid: string;
-  search?: {query: string | null, selectedResult: SelectedResult | null};
+  search?: {query: string | null};
```

Comment:
I'm almost sure that this is no longer required because I no longer use location.state to retrieve data from, I just didn't check it since it's still WIP. Please leave this conversation unresolved, so I can go back to this later

Reply:
yea i think this can be removed right?